pm/recipient.go

```diff
@@ -274,10 +274,10 @@ func (r *recipient) acceptTicket(ticket *Ticket, sig []byte, recipientRand *big.
 		// When a winning ticket is redeemed, the ticket's recipientRand is invalidated
 		// and the sender must send tickets with a new seed, but there could be a delay
 		// before the sender is notified of the new seed.
-		//
-		// TODO(yondonfu): Track the number of these types of errors to determine
-		// whether we should accept more of these types of errors
-		return errors.Errorf("invalid already revealed recipientRand %v", recipientRand)
+		return newReceiveError(
+			errors.Errorf("invalid already revealed recipientRand %v", recipientRand),
```

Comment:
I think there's a word missing here ? 
eg. 
`"Invalid seed, already revealed reciepientRand %v"`
or
`"Invalid ticket, already revealed reciepientRand %v"`

Reply:
I feel like this error message does accurately describe what went wrong while keeping it concise, but happy to change it if you feel strongly about it

Reply:
No not at all. I just reads a bit weird that's all, perhaps a colon or comma might help but up to you.